test/express/test-simple-server-custom-formatters.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const inputValidation = require('../../src/middleware');
+
+function customFormatter(errors, config) {
+    return {
+        errors: JSON.stringify(errors),
+        isCustom: true,
+        extraText: config.paramsForFormatter.extraText
+    };
+}
+
+module.exports = inputValidation.init('test/pet-store-swagger.yaml',
+    {
+        errorFormatter: customFormatter,
+        paramsForFormatter: {
+            extraText: 'dummy-text'
+        }
+    })
+    .then(() => {
+        const app = express();
+        app.use(bodyParser.json());
+        app.post('/pets', inputValidation.validate, (req, res, next) => {
+            res.json({ result: 'OK' });
+        });
+        app.use((err, req, res, next) => {
+            if (err.isCustom) {
```

Comment:
@kibertoad i'm not sure this condition is necessary here.

whenever there would be a bug like the custom formatter isn't being executed, the app would hang the response and the test would fail on a timeout, instead of just returning the wrong error formatting.

how about just returning the error object in the response?

Reply:
Done!